do/serverless.go

```diff
@@ -296,13 +324,25 @@ func NewServerlessService(client *godo.Client, usualServerlessDir string, credsT
 		userAgent:     fmt.Sprintf("doctl/%s serverless/%s", doctl.DoitVersion.String(), minServerlessVersion),
 		client:        client,
 		owClient:      nil,
+		accessToken:   accessToken,
 	}
 }
 
+// HashAccessToken converts a DO access token string into a shorter but suitably random string
+// via hashing.  This is used to form part of the path for storing OpenWhisk credentials
+func HashAccessToken(token string) string {
+	hasher := sha1.New()
```

Comment:
Relocated from the `commands` package to the `do` package to avoid having to pass both the token and its hash at init time.